src/schema/relations/type-helpers.ts

```
@@ -1,8 +1,14 @@
 import type { ObjectId } from "mongodb";
-import type { Index } from "../../type-helpers";
+import { Limit, Skip, Sort } from "../../collection/types/pipeline-stage";
+import {
+  BoolProjection,
+  WithProjection,
+} from "../../collection/types/query-options";
```

Biome (1.9.4)

[error] All these imports are only used as types.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Use import type.


(lint/style/useImportType)